packages/core/test/fixtures/transaction.ts

```diff
@@ -293,6 +303,9 @@ export const validTransactions = [
                     script: '76a914851a33a5ef0d4279bd5854949174e2c65b1d450088ac',
                 },
             ],
+            inPubkeys: [
+                '21038de63cf582d058a399a176825c045672d5ff8ea25b64d28d4375dcdb14c02b2bac',
```

Comment:
this is also not a pub key

Reply:
This was taken from a P2SH address and this test has been skipped, removed the pubkey test data as its misleading